src/main/java/com/google/cloud/resourcemanager/spi/v1beta1/HttpResourceManagerRpc.java

```diff
@@ -301,4 +301,25 @@ public Policy replacePolicy(String projectId, Policy newPolicy) throws ResourceM
       throw translate(ex);
     }
   }
+
+  @Override
+  public List<Boolean> testOrgPermissions(String resource, List<String> permissions) {
```

Comment:
This API is strange. One rarely encounters a list of booleans. What does someone do with it? How do you know which boolean maps to which permission?